appveyor.yml

```diff
@@ -1,60 +1,21 @@
 ---
-version: "{build}"
-clone_depth: 10
-init:
-  - set PATH=C:\ruby%ruby_version%\bin;C:\Program Files\7-Zip;C:\Program Files\AppVeyor\BuildAgent;C:\Program Files\Git\cmd;C:\Windows\system32
-  - if %ruby_version%==_trunk (
-      appveyor DownloadFile https://ci.appveyor.com/api/projects/MSP-Greg/ruby-loco/artifacts/ruby_trunk.7z -FileName C:\ruby_trunk.7z &
-      7z x C:\ruby_trunk.7z -oC:\ruby_trunk &
-      C:\ruby_trunk\trunk_pkgs.cmd
-    )
 install:
-  # RI DevKit is only installed in Ruby23 and Ruby23-x64 folders
-  # need to install OpenSSL since standard DevKit doesn't contain it
-  - if "%ri_file%"=="x86" (
-      appveyor DownloadFile https://dl.bintray.com/oneclick/OpenKnapsack/x86/openssl-1.0.2j-x86-windows.tar.lzma &
-      7z e openssl-1.0.2j-x86-windows.tar.lzma &
-      7z x -y openssl-1.0.2j-x86-windows.tar -oC:\ruby23\DevKit\mingw &
-      set b_config="--with-ssl-dir=C:/ruby23/DevKit/mingw --with-opt-include=C:/ruby23/DevKit/mingw/include" &
-      set SSL_CERT_FILE=C:/ruby24-x64/ssl/cert.pem
-    )
-  - if "%ri_file%"=="x64" (
-      appveyor DownloadFile https://dl.bintray.com/oneclick/OpenKnapsack/x64/openssl-1.0.2j-x64-windows.tar.lzma &
-      7z e openssl-1.0.2j-x64-windows.tar.lzma &
-      7z x -y openssl-1.0.2j-x64-windows.tar -oC:\ruby23-x64\DevKit\mingw &
-      set b_config="--with-ssl-dir=C:/ruby23-x64/DevKit/mingw --with-opt-include=C:/ruby23-x64/DevKit/mingw/include" &
-      set SSL_CERT_FILE=C:/ruby24-x64/ssl/cert.pem
-    )
-  - ruby --version
-  - gem  --version
-  - bundler --version
-  - bundle install --without documentation --path vendor/%ruby_version%/bundle
+  # download shared script files
+  - ps: >-
+      if ( !(Test-Path -Path ./shared -PathType Container) ) {
+        $uri = 'https://ci.appveyor.com/api/projects/MSP-Greg/av-gem-build-test/artifacts/shared.7z'
```

Comment:
This may not be hosted indefinitely. Is there another canonical source? I can set up another repo under github.com/eventmachine just to host these as Releases, for example.

Reply:
> This may not be hosted indefinitely

In the future I suppose I could win the lottery...

Maybe you could fork it and run an Appveyor build on it to create the 7z file?

An example of why I thought it best that you use my repo is how about updates to OpenSSL packages?  I can go into the repo, update the package urls, then test my fork of EM,   Of course, I'd post an issue or contact you to let you know of the change.